src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeUnknown48hRiskCellConfigurator.swift

```diff
@@ -0,0 +1,112 @@
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import UIKit
+
+final class HomeUnknown48hRiskCellConfigurator: HomeRiskLevelCellConfigurator {
+
+	private var previousRiskLevel: EitherLowOrIncreasedRiskLevel?
+
+	// MARK: Creating a unknown 48h Risk cell
+	init(
+		isLoading: Bool,
+		lastUpdateDate: Date?,
+		detectionInterval: Int,
+		detectionMode: DetectionMode,
+		manualExposureDetectionState: ManualExposureDetectionState?,
+		previousRiskLevel: EitherLowOrIncreasedRiskLevel?
+	) {
+		self.previousRiskLevel = previousRiskLevel
+		super.init(
+			isLoading: isLoading,
+			isButtonEnabled: manualExposureDetectionState == .possible,
+			isButtonHidden: detectionMode == .automatic,
+			detectionIntervalLabelHidden: detectionMode != .automatic,
+			lastUpdateDate: lastUpdateDate,
+			detectionInterval: detectionInterval
+		)
+	}
+
+	override func configure(cell: RiskLevelCollectionViewCell) {
+		cell.delegate = self
+
+		let title: String = isLoading ? AppStrings.Home.riskCardStatusCheckTitle : AppStrings.Home.riskCardUnknownTitle
+		let titleColor: UIColor = .enaColor(for: .textContrast)
+		cell.configureTitle(title: title, titleColor: titleColor)
+
+		let body: String = AppStrings.Home.riskCardUnknown48hBody
+		cell.configureBody(text: body, bodyColor: titleColor, isHidden: isLoading)
+
+		let color: UIColor = .enaColor(for: .riskNeutral)
+		let separatorColor: UIColor = .enaColor(for: .hairlineContrast)
+		var itemCellConfigurators: [HomeRiskViewConfiguratorAny] = []
+		if isLoading {
+			let isLoadingItem = HomeRiskLoadingItemViewConfigurator(title: AppStrings.Home.riskCardStatusCheckBody, titleColor: titleColor, isLoading: true, color: color, separatorColor: separatorColor)
+			itemCellConfigurators.append(isLoadingItem)
+		} else {
+
+			let previousRiskTitle: String
+			switch previousRiskLevel {
+			case .low?:
+				previousRiskTitle = AppStrings.Home.riskCardLastActiveItemLowTitle
+			case .increased?:
+				previousRiskTitle = AppStrings.Home.riskCardLastActiveItemHighTitle
+			default:
+				previousRiskTitle = AppStrings.Home.riskCardLastActiveItemUnknownTitle
+			}
+
+			let activateItemTitle = String(format: AppStrings.Home.riskCardLastActiveItemTitle, previousRiskTitle)
+			let iconTintColor: UIColor = titleColor
+			let item1 = HomeRiskImageItemViewConfigurator(title: activateItemTitle, titleColor: titleColor, iconImageName: "Icons_LetzteErmittlung-Light", iconTintColor: iconTintColor, color: color, separatorColor: separatorColor)
+			let dateTitle = String(format: AppStrings.Home.riskCardDateItemTitle, lastUpdateDateString)
+			let item2 = HomeRiskImageItemViewConfigurator(title: dateTitle, titleColor: titleColor, iconImageName: "Icons_Aktualisiert", iconTintColor: iconTintColor, color: color, separatorColor: separatorColor)
+			itemCellConfigurators.append(contentsOf: [item1, item2])
+
+		}
```

Comment:
i believe all manipulations to the itemCellConfigurators can be extracted or moved to methods or simplified 💯